docs/cep/query-guide/functions/core/maximum.md

```diff
@@ -6,20 +6,25 @@ Returns the maximum value of the input parameters.
 
 ## Syntax
 
-    <INT|LONG|DOUBLE|FLOAT> maximum(<INT|LONG|DOUBLE|FLOAT> arg, <INT|LONG|DOUBLE|FLOAT> ...)
+```sql
+<INT|LONG|DOUBLE|FLOAT> maximum(<INT|LONG|DOUBLE|FLOAT> arg, <INT|LONG|DOUBLE|FLOAT> ...)
+```
 
 ## Query Parameters
 
 | Name | Description   | Default Value | Possible Data Types   | Optional | Dynamic |
-|------|---------------------------------------------------|------------|----------------|----------|---------|
+|------|----------------|------------|----------------|----------|---------|
 | arg  | This function accepts one or more parameters. They can belong to any one of the available types. All the specified parameters should be of the same type. |          | INT LONG DOUBLE FLOAT | No       | Yes     |
 
 ## Example
 
 ```sql
-@info(name = 'query1') FROM inputStream
+@info(name = 'query1')
+FROM inputStream
```

Comment:
FROM inputStream needs to be at the end, i.e. after SELECT statement

Reply:
Wow, I apparently completely screwed up this PR...sorry guys! I thought I had done this on a separate branch.